README.md

```diff
@@ -125,3 +131,5 @@ yarn startd
 Application then runs on http://localhost:3001 by default.
 The port can be changed overriding the `local.json` file,
 however, any client will also need to be updated.
+
+To test that things are running correctly, navigate to your [local oath link](localhost:3001/google/oauth) and after you select an account, you should see a log in your yarn console that your account was created.
```

Comment:
this is fine... but we should have a better test. Some sort of health check I guess.

Reply:
Sure sure. Just putting in what we have now, if only to force us to fix the "Cannot GET /google/callback" issue I saw while doing this (unless you want me to submit an issue :P)